scripts/variantstore/beta_docs/gvs-quickstart.md

```diff
@@ -148,8 +156,4 @@ If you use plan to publish data analyzed using the GVS workflow, please cite the
 
 Details on citing Terra workspaces can be found here: [How to cite Terra](https://support.terra.bio/hc/en-us/articles/360035343652)
 
-Data Sciences Platform, Broad Institute (*Year, Month Day that the workspace was last modified*) gvs-prod/Genomic_Variant_Store_Beta [workspace] Retrieved *Month Day, Year that workspace was retrieved*, https://app.terra.bio/#workspaces/gvs-prod/Genomic_Variant_Store_Beta
-
-### License
-**Copyright Broad Institute, 2022 | BSD-3**  
```

Comment:
Why is this removed?

Reply:
Removed because the license info was for WARP

Reply:
Is there something we should replace that with?

Reply:
Technically bc the code is in GATK repo it's currently under that license: https://github.com/broadinstitute/gatk/blob/master/LICENSE.TXT

Let's use that for now.

Reply:
Added GATK license